StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Saved Payment Method Screen/SavedPaymentMethodCollectionView.swift

```diff
@@ -43,13 +45,22 @@ class SavedPaymentMethodCollectionView: UICollectionView {
     }
 
     var isRemovingPaymentMethods: Bool = false
+    var showDefaultPMBadge: Bool = false
```

Comment:
This should be a `let` like `appearance`, and we shouldn't need to define `false` as it's default value twice.